verilog/CST/verilog_nonterminals.h

```diff
@@ -296,7 +296,13 @@ enum class NodeEnum {
   kContinuousAssign,
   kLoopGenerateConstruct,
   kConditionalGenerateConstruct,
+  kConditionalGenerateConstructCase,
```

Comment:
Suggest naming this: `kCaseGenerateConstruct` to resemble other things starting with `kCase`

Reply:
Done